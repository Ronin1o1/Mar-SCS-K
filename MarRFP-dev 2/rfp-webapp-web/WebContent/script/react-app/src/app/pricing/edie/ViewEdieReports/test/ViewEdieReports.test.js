import React from "react";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import ViewEdieReports from "../content/ViewEdieReports";

it("renders without crashing", () => {
  mount(<ViewEdieReports />);
});

it("renders correctly", () => {
  const tree = mount(<ViewEdieReports />);
  expect(toJson(tree)).toMatchSnapshot();
});

