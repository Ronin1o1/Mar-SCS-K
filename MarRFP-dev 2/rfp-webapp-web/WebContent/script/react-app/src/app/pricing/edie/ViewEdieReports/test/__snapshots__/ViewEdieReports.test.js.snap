// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RequestSpecialReports>
  <ViewEdieReportsContextProvider>
    <table
      className="fullHeight"
    >
      <tr>
        <td />
      </tr>
    </table>
  </ViewEdieReportsContextProvider>
</RequestSpecialReports>
`;
